src/directives/partials/scaleselector.html

```diff
@@ -1,12 +1,14 @@
-<div ng-class="::{'dropdown': true, 'dropup': scaleselectorCtrl.options.dropup}">
-  <button type="button" class="btn btn-primary" data-toggle="dropdown">
-    <span ng-bind-html="scaleselectorCtrl.currentScale"></span>&nbsp;<span class="caret"></span>
-  </button>
-  <ul class="dropdown-menu" role="menu">
-    <li ng-repeat="zoomLevel in ::scaleselectorCtrl.zoomLevels">
-      <a href ng-click="scaleselectorCtrl.changeZoom(zoomLevel)"
-         ng-bind-html="scaleselectorCtrl.getScale(zoomLevel)">
-      </a>
-    </li>
-  </ul>
+<div class="form-group form-group-sm dropdown" ng-class="::{'dropup': scaleselectorCtrl.options.dropup}">
+  <div class="btn-group btn-block">
+      <button type="button" class="btn btn-default form-control" data-toggle="dropdown" aria-expanded="false">
```

Comment:
You need to add the `dropdown-toggle` class name otherwise the corners are squared on the right.
![screenshot from 2016-06-08 11 48 45](https://cloud.githubusercontent.com/assets/319774/15890259/fca57596-2d6e-11e6-83a9-b7e85b64570e.png)